test/functional/unified-spec-runner/operations.ts

```diff
@@ -230,8 +234,13 @@ operations.set('endSession', async ({ entities, operation }) => {
 
 operations.set('find', async ({ entities, operation }) => {
   const collection = entities.getEntity('collection', operation.object);
+  const session = entities.getEntity('session', operation.arguments.session, false);
   const { filter, sort, batchSize, limit } = operation.arguments;
-  return collection.find(filter, { sort, batchSize, limit }).toArray();
+  const cursor = collection.find(filter, { sort, batchSize, limit, session });
+  if (session) {
```

Comment:
If a unified test specifies a named session in the `arguments`, we need to set it on the cursor.